From 6fd29a6889754de45455bf2eb889e7e8f64dda15 Mon Sep 17 00:00:00 2001
From: kbandi <kushal.bandi@intel.com>
Date: Sun, 17 Jun 2018 23:40:22 -0700
Subject: [PATCH 1547/2328] media: intel-ipu4: css:
 scci_stable_iot_mr4_beta_20180506_20180529_1638

    Message for Open Source:
    New firmware css release.

    Message for Internal:
    [Feature]New firmware css release.
    [Root Cause/Changes]
    isys & psys package lib2600b0 for commit-id 1cf69aa
    FW_Manifest_ID: 53927e731bc24b7ba2803ab4577d7307dabac0de
    FW_Build_Branch: stable_iot_mr4_beta_20180506
    FW_Build_Topic: auto_merge_queue
    library version: commit-id 1cf69aa
    FW Bundle patch number of this release :  332146
    FW tag : scci_stable_iot_mr4_beta_20180506_20180529_1638

Change-Id: Ie9934d620e2acd8c7c2a9b8ee4dbb6523c54765b
Signed-off-by: kbandi <kushal.bandi@intel.com>
---
 drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h           | 2 +-
 .../lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h        | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h b/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
index 795c783..eea56e7 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
@@ -11,4 +11,4 @@
 * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
 * more details.
 */
-#define IA_CSS_FW_PKG_RELEASE  0x20180420
+#define IA_CSS_FW_PKG_RELEASE  0x20180529
diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
index 795c783..eea56e7 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
@@ -11,4 +11,4 @@
 * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
 * more details.
 */
-#define IA_CSS_FW_PKG_RELEASE  0x20180420
+#define IA_CSS_FW_PKG_RELEASE  0x20180529
-- 
2.7.4

